libs/ui-watt/src/lib/components/input/date-range-input/watt-date-range-input.component.ts

```diff
@@ -0,0 +1,268 @@
+  ) {
+    this.parentControlDirective.valueAccessor = this;
+  }
+
+  /**
+   * @ignore
+   */
+  ngAfterViewInit() {
+    if (this.initialValue) {
+      this.writeValue(this.initialValue);
+    }
+
+    // Setup input masks
+    const startDateInputElement = this.startDateInput.nativeElement;
+    const startDateInputMask = this.inputMaskService.mask(
+      this.inputFormat,
+      this.placeholder,
+      startDateInputElement,
+      this.onBeforePaste
+    );
+
+    const endDateInputElement = this.endDateInput.nativeElement;
+    const endDateInputMask = this.inputMaskService.mask(
+      this.inputFormat,
+      this.placeholder,
+      endDateInputElement,
+      this.onBeforePaste
+    );
+
+    // Setup and subscribe for input changes
+    this.rangeInputService.init({
+      startInput: {
+        element: startDateInputElement,
+        initialValue: this.initialValue?.start,
+        mask: startDateInputMask,
+      },
+      endInput: {
+        element: endDateInputElement,
+        initialValue: this.initialValue?.end,
+        mask: endDateInputMask,
+      },
+    });
+
+    this.rangeInputService.onInputChanges$
+      ?.pipe(takeUntil(this.destroy$))
+      .subscribe(([start, end]) => {
+        this.markParentControlAsTouched();
+        this.changeParentValue({ start, end });
+      });
+  }
+
+  /**
+   * @ignore
+   */
+  ngOnDestroy() {
+    this.destroy$.next();
+    this.destroy$.complete();
+  }
+
+  /**
+   * @ignore
+   */
+  writeValue(dateRange: WattDateRange | null): void {
+    if (!this.startDateInput || !this.endDateInput) {
+      this.initialValue = dateRange;
+      return;
+    }
+
+    const inputEvent = new Event('input', { bubbles: true });
+
+    if (dateRange?.start) {
+      this.startDateInput.nativeElement.value = dateRange.start;
+      this.startDateInput.nativeElement.dispatchEvent(inputEvent);
+    }
+
+    if (dateRange?.end) {
+      this.endDateInput.nativeElement.value = dateRange.end;
+      this.endDateInput.nativeElement.dispatchEvent(inputEvent);
+    }
+  }
+
+  /**
+   * @ignore
+   */
+  registerOnChange(onChangeFn: (value: WattDateRange) => void): void {
+    this.changeParentValue = onChangeFn;
+  }
+
+  /**
+   * @ignore
+   */
+  registerOnTouched(onTouchFn: () => void) {
+    this.markParentControlAsTouched = onTouchFn;
+  }
+
+  /**
+   * @ignore
+   */
+  setDisabledState(isDisabled: boolean): void {
+    this.isDisabled = isDisabled;
+    this.changeDetectorRef.detectChanges();
+  }
+
+  /**
+   * @ignore
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  private changeParentValue = (value: WattDateRange): void => {
+    // Intentionally left empty
+  };
+
+  /**
+   * @ignore
+   */
+  private markParentControlAsTouched = (): void => {
+    // Intentionally left empty
+  };
+
+  /**
+   * @ignore
+   */
+  private onBeforePaste(pastedValue: string): string {
+    if (this.locale !== 'da') return pastedValue;
```

Comment:
suggestion: We have the `danish-locale-code.ts` file but it's unavailable in this library. So we can't reuse it. Can we at least extract the `da` value to its own variable at the top of the file. It's already repeated a couple of times.